jdaviz/configs/cubeviz/plugins/slice/slice.py

```diff
@@ -55,14 +55,19 @@ def _on_linked_changed(self, event):
                                           self._viewer_slices_changed)
 
     def _on_data_added(self, msg):
-        if len(msg.data.shape) == 3 and isinstance(msg.viewer, BqplotImageView):
-            self.max_value = msg.data.shape[0] - 1
+        if isinstance(msg.viewer, BqplotImageView):
+            if len(msg.data.shape) == 3:
+                self.max_value = msg.data.shape[-1] - 1
```

Comment:
for the example notebook, this seems to incorrectly set a maximum of 73 slices (the slider does not allow scrubbing through the whole cube).

Reply:
Do you have the required `glue-astronomy` version? The slice plugin works fine for me.

Reply:
doh!  Forgot I had downgraded to test something else recently.  Sorry about that!